bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/api/ViessmannAuth.java

```diff
@@ -256,18 +255,18 @@ private void getRefreshTokens() throws ViessmannAuthException {
 
     private void updateBridgeStatusLogin() {
         bridgeHandler.updateBridgeStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
-                String.format("Login fails. Please check user and password."));
+                "Login fails. Please check user and password.");
     }
 
     private void updateBridgeStatusApiKey() {
         bridgeHandler.updateBridgeStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
-                String.format("Login fails. Please check API Key."));
+                "Login fails. Please check API Key.");
     }
 
-    private @Nullable String executeUrlAuthorize(String method, String url) {
+    private @Nullable String executeUrlAuthorize(String url) {
         Request request = httpClient.newRequest(url);
         request.timeout(API_TIMEOUT_MS, TimeUnit.MILLISECONDS);
-        request.method(method);
+        request.method("GET");
```

Comment:
You can use predefined const `HttpMethod.GET` here. Furthermore the `Request` can be used like a builder pattern.

For example:
```suggestion
        Request request = httpClient.newRequest(...).timeout.(...).method(...).header(..., ...);
```